docs/precommit.md

```diff
@@ -15,7 +15,7 @@ _Make sure Prettier is installed and is in your [`devDependencies`](https://docs
 npx mrm lint-staged
 ```
 
-This will install [simple-git-hooks](https://github.com/toplenboren/simple-git-hooks) and [lint-staged](https://github.com/okonet/lint-staged), then add a configuration to the project’s `package.json` that will automatically format supported files in a pre-commit hook.
+This will install [husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged), then add a configuration to the project’s `package.json` that will automatically format supported files in a pre-commit hook.
```

Comment:
This seems to overwrite recent PR #10601.

I would favor documenting lint-staged with Husky as a separate option or sub-option.

Reply:
`mrm-task-lint-staged` now installs husky 6, so this PR reflects only the current state.
https://github.com/sapegin/mrm/compare/mrm-task-lint-staged@4.0.0...mrm-task-lint-staged@5.0.0